lib/plugins/features/movies/controller.js

```diff
@@ -5,6 +5,7 @@ const Location = require('../../../models/location');
 const Movie    = require('../../../models/movie');
 
 exports.create = (payload) => {
+  payload.name = payload.title;
```

Comment:
You may also have to `delete payload.title`. Setting it to undefined might work too. It's not required at this point but if you delete it then you'll be testing that title can now be null